src/musicbrainz/web/musicbrainzrecordingstask.cpp

```diff
@@ -82,8 +85,8 @@ MusicBrainzRecordingsTask::MusicBrainzRecordingsTask(
     // Related Bug: https://bugs.launchpad.net/mixxx/+bug/1983204
     // In order to not hit the rate limits and respect their rate limiting rule.
     // We are going to delay every request by one second.
-    m_timer.start(m_DelayDurationMilis);
-    connect(&m_timer,
+    m_inhibitTimer.start();
```

Comment:
Why you start the timer here?

Reply:
Ah I see, you have picked a bad name. This is the measurement timer that watches the cycle time. It does not inhibit something. This was my propose name for the other timer, but m_requestTimer is perfectly OK.

Reply:
m_requestTimer.setSingleShot(true) is missing, to avoid that his timer fires over and over again.

Reply:
Thank you, I've added that SetSingleShot and naming changed.